tests/config_test.py

```diff
@@ -120,6 +120,24 @@ def test_parse_yaml_document_to_dict_missing_required_key(self):
             config.parse_yaml_document_to_dict,
             yaml_doc)
 
+    def test_load_from_dict(self):
+        config_data = {
+              'developer_token': self.developer_token,
+              'client_id': self.client_id,
+              'client_secret': self.client_secret,
+              'refresh_token': self.refresh_token
+          }
```

Comment:
Indent these lines by 4 spaces and bring the curly bracket up to the line above, like: 

```python
config_data = {
    'developer-token': self.developer_token,
    'client_id': self.client_id,
    'client_secret': self.client_secret,
    'refresh_token': self.refresh_token}
```